__docs__/wonder-blocks-tooltip/tooltip.stories.tsx

```diff
@@ -43,9 +43,12 @@ export default {
         chromatic: {delay: 500},
     },
     decorators: [
-        (Story): React.ReactElement => (
-            <View style={styles.storyCanvas}>{Story()}</View>
-        ),
+        (Story, {parameters}): React.ReactElement =>
+            parameters.layout === "fullscreen" ? (
+                Story()
+            ) : (
+                <View style={styles.storyCanvas}>{Story()}</View>
+            ),
     ],
```

Comment:
**praise:** Neat approach!

Reply:
This was because even if the layout parameter was set to `fullscreen` for a story, the overall decorator for the stories in the file would still get applied and add additional spacing around the story. I couldn't figure out how to opt out of the decorator for one story so I added this condition!